core/src/main/scala/cats/data/StateT.scala

```diff
@@ -1,75 +1,90 @@
 package cats
 package data
 
+import cats.functor.{ Contravariant, Profunctor }
 import cats.syntax.either._
 
 /**
- * `StateT[F, S, A]` is similar to `Kleisli[F, S, A]` in that it takes an `S`
- * argument and produces an `A` value wrapped in `F`. However, it also produces
- * an `S` value representing the updated state (which is wrapped in the `F`
- * context along with the `A` value.
```

Comment:
Can we move the old `StateT` documentation to the `StateT` type alias in _package.scala_?